server/src/main/java/org/opensearch/index/remote/RemoteRefreshSegmentTracker.java

```diff
@@ -153,6 +190,12 @@ public class RemoteRefreshSegmentTracker {
      */
     private final Object uploadBytesMutex = new Object();
 
+    /**
+     * Provides moving average over the last N total size in bytes of segment files downloaded from the remote store.
+     * N is window size. Wrapped with {@code AtomicReference} for dynamic changes in window size.
+     */
+    private final AtomicReference<MovingAverage> downloadBytesMovingAverageReference;
```

Comment:
AtomicReference is not required. We can move this to volatile. Same as applicable for other moving average fields as well.

Reply:
Lets also update the description about the concerned fields.

Reply:
Updated for all download average references